src/components/schedule-visualizer/index.tsx

```diff
@@ -320,19 +320,20 @@ export default function ScheduleVisualizer(props: ScheduleVisualizerProps): Reac
           </LayersControl.Overlay>
 
           <LayersControl.Overlay name="Negotiation Trajectories" checked>
-          {curMapFloorLayer && props.negotiationTrajStore[curMapFloorLayer.level.name] && (
-            <Pane>
-              <RobotTrajectoryContext.Provider value={RobotTrajContextValue}>
-                <RobotTrajectoriesOverlay
-                  bounds={curMapFloorLayer.bounds}
-                  trajs={props.negotiationTrajStore[curMapFloorLayer.level.name].values}
-                  conflicts={getConflicts(curMapFloorLayer.level.name)}
-                  colorManager={colorManager}
-                  conflictRobotNames={conflictRobotNames}
-                />
-              </RobotTrajectoryContext.Provider>
-            </Pane>
-          )}
+            {curMapFloorLayer && (
```

Comment:
Maybe you can check here the existence of `props.negotiationTrajStore[curMapFloorLayer.level.name]` and we can keep the `RobotTrajectoriesOverlay` with `traj` as required.

E.g.:` curMapFloorLayer && props.negotiationTrajStore[curMapFloorLayer.level.name] && ....
`


Also, I think there is a `curLevelName` that you can use instead of `curMapFloorLayer.level.name`. To do something like `props.negotiationTrajStore[curLevelName]`

Reply:
Just tell if I'm missing something here

Reply:
i removed it so the checkbox doesn't unknowingly appear upon clicking a negotiation status TreeItem, causing some UI inconsistency. See 199c136a9289ac3a0eeb52f08dd1dec13571c0f4

replaced with curLevelName 36d816d